website/docs/r/lb.html.markdown

```diff
@@ -38,7 +39,24 @@ resource "aws_lb" "test" {
 
 ```hcl
 # Create a new network load balancer
+resource "aws_lb" "test" {
+  name               = "test-lb-tf"
+  internal           = false
+  load_balancer_type = "network"
+  subnets            = ["${aws_subnet.public.*.id}"]
+
+  enable_deletion_protection = true
 
+  access_logs {
```

Comment:
I don't think NLBs support access logs.

Reply:
They do not, this configuration should be removed

Reply:
fixed